blivet/devices/cache.py

```diff
@@ -53,12 +53,6 @@ def stats(self):
         :rtype: :class:`CacheStats`
         """
 
-    @abc.abstractproperty
-    def mode(self):
-        """Mode of the cache (writeback/writethrough...)
-        :rtype: str
-        """
-
```

Comment:
Why remove this from the base class? There's not any reasonable value in this new class for it?

Reply:
No, dm-writecache is very simple, there is no mode or metadata.